packages/amplify-frontend-ios/lib/frontend-config-creator.js

```diff
@@ -33,7 +33,7 @@ function getAWSConfigObject(amplifyResources) {
   const { serviceResourceMapping } = amplifyResources;
   const configOutput = {
     UserAgent: 'aws-amplify/cli',
-    Version: '0.1.0',
+    Version: '1.0',
```

Comment:
I think that it would be nice to send the correct version so perhaps metrics from service side could reflect the used CLI version and help with potential troubleshooting, but I reverse it to 0.1.0.